identity-service/scripts/dev-server.sh

```diff
@@ -7,6 +7,7 @@ link_libs=true
 if [ "$link_libs" = true ]
 then
     cd ../audius-libs
+    npm run dev &
```

Comment:
A down won't stop this guy, which is a bit scary. @dylanjeffers what was the plan here? I thought we added something higher-level in service commands that would do this

Reply:
@raymondjacobson since this runs inside the identity container, wouldn't it stop in that context?

Reply:
yeah you're right. that'll work. I do think that maybe it makes sense to run the dev server outside the container since the directory itself is mounted as a volume

Reply:
that makes sense, that would work too

also one other problem i'm facing is apply a similar fix to content node, but i get this error. i think it's because we use nodemon with ts-node but without any transpiling. any ideas here? 

```
+ link_libs=true
+ [[ cn1_creator-node-db_1:5432, cn1_creator-node-redis_1:6379 != '' ]]
+ /usr/bin/wait
Docker-compose-wait starting with configuration:
------------------------------------------------
 - Hosts to be waiting for: [cn1_creator-node-db_1:5432, cn1_creator-node-redis_1:6379]
 - Timeout before failure: 30 seconds 
 - Sleeping time before checking for hosts availability: 0 seconds
 - Sleeping time once all hosts are available: 0 seconds
------------------------------------------------
Checking availability of cn1_creator-node-db_1:5432
Host cn1_creator-node-db_1:5432 is now available
Checking availability of  cn1_creator-node-redis_1:6379
Host  cn1_creator-node-redis_1:6379 is now available
+ [[ -z '' ]]
+ [[ -n '' ]]
+ '[' -z local-ipfs-node ']'
+ '[' -z cn1_creator-node-redis_1 ']'
+ '[' -z postgres://postgres:postgres@cn1_creator-node-db_1:5432/audius_creator_node ']'
+ [[ false == \t\r\u\e ]]
+ [[ true == \t\r\u\e ]]
+ '[' true = true ']'
+ cd ../audius-libs
+ npm link
npm WARN @terra-dev/chrome-extension@2.5.3 requires a peer of react-dom@^17.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @terra-dev/readonly-wallet-modal@2.5.3 requires a peer of react-dom@^17.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @terra-dev/walletconnect-qrcode-modal@2.5.3 requires a peer of react-dom@^17.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN styled-components@5.3.3 requires a peer of react-dom@>= 16.8.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 1291 packages in 8.586s

156 packages are looking for funding
  run `npm fund` for details

found 66 vulnerabilities (6 low, 13 moderate, 47 high)
  run `npm audit fix` to fix them, or `npm audit` for details
/usr/lib/node_modules/@audius/libs -> /usr/src/audius-libs
+ cd ../app
+ npm link @audius/libs
/usr/src/app/node_modules/@audius/libs -> /usr/lib/node_modules/@audius/libs -> /usr/src/audius-libs
+ npx nodemon --exec 'node --inspect=0.0.0.0:${debuggerPort} --require ts-node/register src/index.ts'
+ ./node_modules/.bin/bunyan
+ tee /dev/fd/63
++ logger
[nodemon] 1.19.4
[nodemon] to restart at any time, enter `rs`
[nodemon] watching dir(s): *.*
[nodemon] watching extensions: js,ts
[nodemon] starting `node --inspect=0.0.0.0:${debuggerPort} --require ts-node/register src/index.ts`
Debugger listening on ws://0.0.0.0:9230/c850d368-37b7-4d3a-b1e8-6052ffa3c333
For help, see: https://nodejs.org/en/docs/inspector
Warning: configuration param 'awsBucket' not declared in the schema
configuration param 'awsAccessKeyId' not declared in the schema
configuration param 'awsSecretAccessKey' not declared in the schema
[2022-03-23T00:42:31.048Z]  INFO: audius_creator_node/52 on 328e7e6a68e2: Loglevel set to: debug
DM HERE 222
Wed, 23 Mar 2022 00:42:38 GMT sequelize deprecated The logging-option should be either a function or false. Default: console.log at node_modules/sequelize/lib/sequelize.js:196:13
/usr/src/audius-libs/src/utils/multiProvider.ts:1
import { promisify, callbackify } from 'util';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at compileFunction (<anonymous>)
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Module.m._compile (/usr/src/app/node_modules/ts-node/src/index.ts:1371:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Object.require.extensions.<computed> [as .ts] (/usr/src/app/node_modules/ts-node/src/index.ts:1374:12)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.<anonymous> (/usr/src/audius-libs/src/utils/index.js:4:27)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Module.m._compile (/usr/src/app/node_modules/ts-node/src/index.ts:1371:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Object.require.extensions.<computed> [as .js] (/usr/src/app/node_modules/ts-node/src/index.ts:1374:12)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.<anonymous> (/usr/src/audius-libs/src/services/creatorNode/index.js:5:18)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Module.m._compile (/usr/src/app/node_modules/ts-node/src/index.ts:1371:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Object.require.extensions.<computed> [as .js] (/usr/src/app/node_modules/ts-node/src/index.ts:1374:12)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
[nodemon] app crashed - waiting for file changes before starting...
```

Reply:
as for running this inside vs outside the container i'll defer to you two, not as familiar with the ins and outs of TS

Reply:
looking into this

Reply:
looking into content-node issue as well @dmanjunath thanks for flagging

Reply:
Addressed here! https://github.com/AudiusProject/audius-protocol/pull/2745